bittide-instances/data/tcl/HardwareTest.tcl

```diff
@@ -76,52 +98,61 @@ proc print_all_vios {} {
     puts $sep
 }
 
-# Open the hardware manager and connect to the hardware server at the given url.
-# Checks whether the expected number of hardware targets or more are connected,
-# if not exit.
-proc connect_expected_targets {url expected} {
-    open_hw_manager
-    connect_hw_server -url $url
-
-  set start_time [clock milliseconds]
-  set i 0
-  while 1 {
-    # Check is expected number of hardware targets are connected
-    set hw_targets [get_hw_targets -quiet]
-    set hw_target_count [llength $hw_targets]
-    if {$hw_target_count >= $expected} {
-      puts "Hardware server at ${url} hosts ${hw_target_count} hardware targets:"
-      puts "$hw_targets"
-      break
+# Checks if listb is a subset of lista.
```

Comment:
Can you add a warning that this is `O(n^2)` / quadratic complexity? It shouldn't be used for lists that are expected to grow big.

Reply:
How about writing this function as (pseudo):

```python
for elem in listb:
  if elem not in lista:
    return 0
return 1
```

Reply:
Lastly I'd prefer it if you'd swap the arguments. _arg1 is subset of arg2_ instead of _arg2 is subset of arg1_.